plugins/aks-desktop/src/components/ImportAKSProjects/ImportAKSProjects.tsx

```diff
@@ -260,6 +260,14 @@ export default function ImportAKSProjects() {
           continue;
         }
 
+        // add allowed namespaces
+        const settings = JSON.parse(
+          localStorage.getItem(`cluster_settings.${clusterName}`) || '{}'
+        );
```

Comment:
We have `getAllowedNamespaces` which we export to plugins AFAIU. We shouldn't use the actual settings' keys for preventing their eventual change in the future.